doc/content/microreactors/hpmr_h2/hpmr_h2_description.md

```diff
@@ -0,0 +1,54 @@
+
+Four characteristics distinguish this HPMR-H$_2$ model from current VTB microreactors models:
+
+- The neutronics analysis considers the impact of hydrogen redistribution within the yttrium hydride pins. This is achieved by incorporating hydrogen redistribution modeling into Bison [!citep](BISON) and by parameterizing the multigroup macroscopic cross sections based on the stoichiometric ratio.
+- The coupling between neutronics and solid heat conduction, including the 101 heat pipe sub-applications, is achieved through a fixed-point iteration approach. This logic relies on the effective multiplication factor's residual error reduction, deviating from the default coupling method performed at each Richardson iteration. This logics allows to minimize the number of iterations between neutron transport and the other physics, and, therefore, the execution time of the simulation.
+- The heat pipe is modeled using the Vapor-Only Flow Model (VOFM) in Sockeye. This heat pipe model couples "a 1D single-phase, compressible flow formulation for the vapor phase in the core region to 2D heat conduction in the wick and annulus" [!citep](SOCKEYE). The VOFM allows to obtain better accuracy with respect to the Conduction Heat Pipe Model, based on thermal resistance, while being numerically more stable than the two-phase Flow-Model.
+- Special care is taken to obtain excellent mass and energy conservation across the multiphysics model, without relying on mesh refinement. As detailed in [!citep](Terlizzi2023), the total hydrogen mass was conserved within 4e-5% and the total global energy discrepancy is below 0.04%.
+
+!media hpmr_h2/hpmr_h2_geometry.jpeg
+    caption= Overview of the reactor geometry [!citep](Terlizzi2023).
+    id=hpmr_h2_geometry
+    style=width:100%;margin-left:auto; margin-right:auto
+
+## Core Description
+
+The HPMR-H$_2$ is a 2-MW microreactor composed of 18 hexagonal assemblies arranged into two rings (see [hpmr_h2_geometry]). The tops and bottoms of these 160-cm-high assemblies are surrounded by 20-cm-high axial beryllium reflectors. Each assembly contains 96 fuel pins that are 1 cm in radius, 60 $YH_x$ pins that are 0.975 cm in radius, and 61 1-cm-radius sodium heat pipes (HPs) drilled into a graphite monolith. The HPs penetrate only into the top axial reflector, making the reactor axially asymmetric. The central shutdown rod slot is empty. The core is surrounded by 12 control drums, with boron carbide employed as the absorbing material. For a simplified mesh, the beryllium radial reflector is hexagonal. The geometries and material specifications of the HPMR-H$_2$ reactor assembly components are reported in [table-floating1] and [table-floating2].
```

Comment:
```suggestion
The HPMR-H$_2$ is a 2-MW microreactor composed of 18 hexagonal assemblies arranged into two rings (see [hpmr_h2_geometry]). The tops and bottoms of these 160-cm-high assemblies are surrounded by 20-cm-high axial beryllium reflectors. Each assembly contains 96 fuel pins that are 1 cm in radius, 60 $YH_x$ pins that are 0.975 cm in radius, and 101 1-cm-radius sodium heat pipes (HPs) drilled into a graphite monolith. The HPs penetrate only into the top axial reflector, making the reactor axially asymmetric. The central shutdown rod slot is empty. The core is surrounded by 12 control drums, with boron carbide employed as the absorbing material. For a simplified mesh, the beryllium radial reflector is hexagonal. The geometries and material specifications of the HPMR-H$_2$ reactor assembly components are reported in [table-floating1] and [table-floating2].
```
@vincentlaboure ?

Reply:
No, it's 61 per assembly but there are one assembly + 2 1/2 assemblies so 101 HPs total

Reply:
changing it back thanks for checking